jvb/src/main/kotlin/org/jitsi/videobridge/metrics/Metrics.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright @ 2024 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.videobridge.metrics
+
+import org.jitsi.config.JitsiConfig
+import org.jitsi.metaconfig.config
+import org.jitsi.metrics.MetricsUpdater
+import org.jitsi.videobridge.util.TaskPools
+import java.time.Duration
+
+object Metrics {
+    private val interval: Duration by config {
+        "videobridge.stats.interval".from(JitsiConfig.newConfig)
```

Comment:
Reading the stat in more than one place seems odd -- if you want this to be the same as `StatsManagerConfig.interval` why not just reference the object?

Reply:
Ops, the other one was supposed to go away but I forgot it.